CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Fourmolu action v3
```

Comment:
Does this need to get tagged for it to be used as the v3 action? Is there no other ref to v3 in the repo?

Reply:
Yes, it does need to get tagged.  I'll do that after merging!